rasa/nlu/featurizers/sparse_featurizer/lexical_syntactic_featurizer.py

```diff
@@ -254,6 +254,12 @@ def train(self, training_data: TrainingData,) -> Resource:
         """
         feature_to_idx_dict = self._create_feature_to_idx_dict(training_data)
         self._set_feature_to_idx_dict(feature_to_idx_dict=feature_to_idx_dict)
+        if not self._feature_to_idx_dict:
+            rasa.shared.utils.io.raise_warning(
+                "No lexical syntactic features could be extracted from the training"
+                "data. In order for this component to work you need to define "
+                "`features` that can be found in the given training data."
```

Comment:
I think it somewhere in Notion states to use single quotes 🙈 
```suggestion
                "No lexical syntactic features could be extracted from the training "
                "data. In order for this component to work you need to define "
                "'features' that can be found in the given training data."
```

Reply:
I was reading this as code (" When including code inside a docstring, use back-ticks ("`") to surround the text (like you would do in Markdown). "). The examples in notion use ' around e.g. printouts of constants. I guess here I could just remove the ` entirely - but ` should be ok, no?

Reply:
Oops. Backticks are the way to go then 👍🏻